src/services/podcast.ts

```diff
@@ -6,7 +6,9 @@ import config from '~/config'
 const { API_BASE_URL } = config()
 
 export const getPodcastsByQuery = async (query) => {
-  const filteredQuery: any = {}
+  const filteredQuery: any = {
+    maxResults: true
```

Comment:
Usually we do not want `maxResults: true` when `getPodcastsByQuery` is called, so we need to change this so that maxResults is passed in as a property on the query object. Instead of:

```
const filteredQuery: any = {
  maxResults: true
}
```

Change it to:

```
const filteredQuery: any = {
  ...(query.maxResults ? { maxResults: true } : {})
}
```

I know this looks like a funny way to add key/value to an object...

`  ...(query.maxResults ? { maxResults: true } : {})`

But the idea is, if query.maxResults is truthy, then `maxResults: true` will be added to the object. If it is not truthy, then `{}` will be used, which doesn't do anything, so it will still be an empty object.